src/nft/components/bag/BagHeader.tsx

```diff
@@ -25,6 +24,23 @@ const ControlRow = styled.div`
   flex-direction: row;
   justify-content: space-between;
 `
+const IconWrapper = styled.button`
+  background-color: transparent;
+  border-radius: 8px;
+  border: none;
+  color: ${({ theme }) => theme.textPrimary};
+  cursor: pointer;
+  display: flex;
+  padding: 2px;
+  opacity: 1;
+  transition: 125ms ease opacity;
+  :hover {
+    opacity: 0.6;
+  }
+  :active {
+    opacity: 0.4;
+  }
```

Comment:
Food for thought and non-blocking, but since we're using these hover and active states globally, might be worth adding them to the styled library

Reply:
agree